src/features/filter.ts

```diff
@@ -8,5 +8,16 @@ const initialState = {
 export const filterSlice = createSlice({
   name: 'filter',
   initialState,
-  reducers: {},
+  reducers: {
+    setStatus: ({ query }, { payload }: PayloadAction<string>) => ({
+      query,
+      status: payload,
```

Comment:
The setStatus reducer function destructures 'query' from the state but does not include 'status'. This might lead to unintended behavior if 'status' is needed elsewhere in the state.